lib/KTable/index.vue

```diff
@@ -231,6 +224,33 @@
         type: Boolean,
         default: false,
       },
+      /**
+       * Indicates whether the table is to be sorted by default by any header or not. By default it is an empty object which means no default sorting is to be used. It accepts a configuration object `{ columnId, direction }`. `columnId` references a `columnId` defined for a header in `headers`. This specifies a column by which the table should be sorted when initially loaded. `direction` can be `'asc'` for ascending or `'desc'` for descending sort direction.
+       */
+      defaultSort: {
+        type: Object,
+        required: false,
+        default: () => ({}),
+        validator: function(value) {
+          if (Object.keys(value).length === 0) {
+            return true;
+          }
+
+          return (
+            ['columnId', 'direction'].every(key => key in value) &&
+            ['asc', 'desc'].includes(value.direction) &&
+            ['string', 'number'].includes(typeof value.columnId)
+          );
+        },
+      },
+      /**
+       * Disables all the sorting functionality provided by the component. This is useful when you want to define your own sorting logic. Refer to the examples above for more details.
```

Comment:
I think we could keep this aligned with the phrasing chosen for the docs example.
```suggestion
       * Disables built-in sort function. This is useful when you want to define your own sorting logic. Refer to the examples above for more details.
```